integrations/access/opsgenie/client.go

```diff
@@ -49,8 +50,12 @@ const (
 
 	ResolveAlertRequestRetryInterval = time.Second * 10
 	ResolveAlertRequestRetryTimeout  = time.Minute * 2
+
+	UUIDRegexPattern = `^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$`
 )
 
+var uuidRegex = regexp.MustCompile(UUIDRegexPattern)
```

Comment:
We should use a UUID parser to verify UUIDs, not a regex.

Reply:
I don't even know why I didn't think of that in the first place. 🤦 

Replaced regex by UUID parser.